src/osx/Installer.Mac/distribution.xml

```diff
@@ -2,7 +2,7 @@
 <installer-gui-script minSpecVersion="1">
     <title>Git Credential Manager</title>
     <background file="background.png" mime-type="image/png" alignment="bottomleft" scaling="tofit" />
-    <options customize="never" />
+    <options customize="never" hostArchitectures="x86_64,arm64" />
```

Comment:
I realise this was also in the first PR, but is correct?

Reply:
Mmmm no. Looks like the separate distribution files may have been dropped in the midst of a rebase. I will bring them back.

Reply:
No